README.md

```diff
@@ -65,6 +65,13 @@ Here we are adding mint.com to the password store, but more specifically to the
 ```
 $ passgo insert mney/mint.com
 ```
+#### passgo show group/pass-name
+Show is used to display a password in standard out. *Previously it was possible to display a password using `passgo group/pass-name` but this is no longer supported*.
```

Comment:
I believe I got it, didn't I?

https://github.com/ejcx/passgo/commit/8e4efffbb43a07d5d161e3745a2d17a16565023a#diff-04c6e90faac2675aa89e2176d2eec7d8R68